tests/e2e/tests/content-releases/releases-page.spec.ts

```diff
@@ -93,33 +93,58 @@ describeOnCondition(edition === 'EE')('Releases page', () => {
   });
 
   test('A user should be able to perform bulk release on entries', async ({ page }) => {
-    await page.getByRole('link', { name: 'Content Manager' }).click();
-
-    await expect(page).toHaveTitle('Content Manager');
-    await expect(page.getByRole('heading', { name: 'Article' })).toBeVisible();
-    const publishedItems = page.getByRole('gridcell', { name: 'published' });
-    expect(publishedItems).toHaveCount(2);
-    const checkbox = page.getByRole('checkbox', { name: 'Select all entries' });
-
-    // Select all entries to release
-    await checkbox.check();
-    const addToRelease = page.getByRole('button', { name: 'add to release' });
-    await addToRelease.click();
+    await test.step('bulk release', async () => {
+      // Navigate to the releases page
+      await page.getByRole('link', { name: 'Releases' }).click();
+      await page.getByRole('button', { name: 'New release' }).click();
+      await expect(page.getByRole('dialog', { name: 'New release' })).toBeVisible();
+      // Create a new release
+      const newReleaseName = 'The Diamond Dogs';
+      await page.getByRole('textbox', { name: 'Name' }).fill(newReleaseName);
+      // Uncheck default scheduling of a release and save
+      await page.getByRole('checkbox', { name: 'Schedule release' }).uncheck();
+      await page.getByRole('button', { name: 'Continue' }).click();
+      // Wait for client side redirect to created release
+      await page.waitForURL('/admin/plugins/content-releases/*');
+      await expect(page.getByRole('heading', { name: newReleaseName })).toBeVisible();
+
+      // Navigate to the content manager
+      await page.getByRole('link', { name: 'Open the Content Manager' }).click();
+      await expect(page).toHaveTitle('Content Manager');
+      await expect(page.getByRole('heading', { name: 'Article' })).toBeVisible();
+      const publishedItems = page.getByRole('gridcell', { name: 'published' });
+      expect(publishedItems).toHaveCount(2);
+      const checkbox = page.getByRole('checkbox', { name: 'Select all entries' });
+
+      // Select all entries to release
+      await checkbox.check();
+      const addToRelease = page.getByRole('button', { name: 'add to release' });
+      await addToRelease.click();
```

Comment:
```suggestion
      await page.getByRole('button', { name: 'add to release' }).click();
```
I'm not sure there's a benefit separating most of these getBy queries and then asserting / actioning on them?